csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -2351,10 +2383,11 @@ Firstly, the program:
 
 Secondly, the program for all items of:
 
+* `/document/acknowledgments[]/organization` and `/vulernabilities[]/acknowledgments[]/organization`: If more than one cvrf:Organization instance is given, the CVRF CSAF converter converts the first one into the `organization`. In addition the converter outputs a warning that information might be lost during conversion of document or vulnerability acknowledgment.
```

Comment:
```suggestion
* `/document/acknowledgments[]/organization` and `/vulnerabilities[]/acknowledgments[]/organization`: If more than one cvrf:Organization instance is given, the CVRF CSAF converter converts the first one into the `organization`. In addition the converter outputs a warning that information might be lost during conversion of document or vulnerability acknowledgment.
```

Reply:
Found the root cause which will be resolved in the next editor revision.